src/ResourceManagement/ServerManagement/ServerManagement.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("KeyVault.Management.Tests")]
```

Comment:
Borrowed file. Not that a assembly title in a test project was all that valuable :)